src/feedinlib/era5.py

```diff
@@ -271,6 +271,21 @@ def select_area(ds, lon, lat, g_step=0.25):
     return answer
 
 
+def extract_coordinates_from_era5(era5_netcdf_filename):
+    """
+    Extract all coordinates from a er5 netCDf-file and return them as a
+    geopandas.Series
+    """
+    ds = xr.open_dataset(era5_netcdf_filename)
```

Comment:
Is it possible to only read longitude and latitude data needed in this function? Would be good for large datasets.